package.json

```diff
@@ -7,29 +7,29 @@
     "dist"
   ],
   "scripts": {
+    "clean": "del-cli --dot=true \"./dist/**/*\"",
+    "build": "npm run clean && npm run build:compile",
+    "build:compile": "cross-env NODE_ENV=production babel src -d dist --verbose --extensions .js --ignore **/**/*.test.js --source-maps",
+    "build:compile:watch": "cross-env NODE_ENV=development babel src -d dist --verbose --watch --extensions .js --ignore **/**/*.test.js --source-maps",
+    "dev": "npm run clean && npm run build:compile:watch",
```

Comment:
Why have watch and dev commands been added?

Reply:
I merged the two scripts into dev.
`"dev": "npm run clean && npm run build:compile -- --watch",`